### PR TITLE
Add push to Docker hub with TAG on image

### PR DESCRIPTION
**Pull request by antoniofilipovic**

### CODE REVIEW

File: .github/workflows/docker_publish.yml [hunk 1]
```diff
@@ -1,22 +1,47 @@
...
+            password: ${{ secrets.DOCKER_PASSWORD }}
+
+      - name: Push production image to Docker Hub
+        id: docker-build-prod
         uses: docker/build-push-action@v2
         with:
-          push: true
-          tags: memgraph/memgraph-mage:latest
\ No newline at end of file
+          push: false
```
Comment (antoniofilipovic) [inline .github/workflows/docker_publish.yml:32]: This is false for now, will change once it is ready for master.

**Review by antoniofilipovic — COMMENTED** [review]
I merged T049 in this branch to check out if everything works with two stages of build, and it does 🎉

**Comment by jmatak** [comment]
When we are about to build the `memgraph-mage:latest`?

**Comment by antoniofilipovic** [comment]
> When we are about to build the `memgraph-mage:latest`?

This was problem, now it is fixed.

File: .github/workflows/docker_publish.yml [hunk 2]
```diff
@@ -1,22 +1,68 @@
...
+          file: ./Dockerfile
+
+      - name: Push development image to Docker Hub
+        id: docker-build-dev
         uses: docker/build-push-action@v2
         with:
-          push: true
-          tags: memgraph/memgraph-mage:latest
\ No newline at end of file
+            push: false
```
Comment (jmatak) [inline .github/workflows/docker_publish.yml:65]: Don't forget to include development image as a part of push to Docker Hub

**Review by jmatak — APPROVED** [review]
:+1:

**Review by jmatak — APPROVED** [review]
:+1:

File: .github/workflows/docker_publish.yml [hunk 3]
```diff
@@ -1,22 +1,68 @@
...
     steps:
       - name: Check out the repo
         uses: actions/checkout@v2
+
+      - name: Get latest tag
+        id: get-latest-tag
+        uses: oprypin/find-latest-tag@v1
+        with:
+          repository: memgraph/mage
+          releases-only: false
```
Comment (jmatak) [inline .github/workflows/docker_publish.yml:21]: @antoniofilipovic Does this need to be true or not?

Comment (jmatak) [inline .github/workflows/docker_publish.yml:21]: @antoniofilipovic Does this need to be true or not?

Comment (jmatak) [reply .github/workflows/docker_publish.yml:21]: Alright, it seems like it is default :+1:

**Pull request merged**